packages/navigation/index.js

```diff
@@ -93,7 +94,13 @@ export const StackNavigatorOptions = {
 export const createTransparentHeaderStyle = (dim: DimensionType) => {
   if (Platform.OS === 'ios' || Device.isWideLayout(dim)) {
     // normal header on iOS and wide Android
-    return {};
+    return {
+      headerStyle: {
+        backgroundColor: 'transparent',
+        marginTop: 20,
+      },
+      headerTransparent: true,
```

Comment:
Is it somehow possible to change the color of the native header (phone signal, battery level, etc.) to be in a different color than black when the image at the top is really dark? And then switch it back to black once you scroll down and enter white areas (or re-instate a non-transparent header?)

Reply:
Yes, it is possible, I didn't notice that detail, but yes, it is set to white on the design, we should definitely change it back to black when we scroll down